### PR TITLE
feat(chat): add styles for chat toolbar

### PR DESCRIPTION
**Pull request by joneff**

### CODE REVIEW

File: packages/default/scss/chat/_layout.scss [hunk 1]
```diff
@@ -414,6 +414,61 @@ $message-group-spacing: calc( #{$chat-avatar-size} + #{$chat-avatar-spacing} );
...
+        }
+        .k-scroll-button-left {
+            left: 0;
+        }
+        .k-scroll-button-right {
+            right: 0;
+        }
+
+        .k-button {
+            border-width: 0;
```
Comment (elena-gancheva) [inline packages/default/scss/chat/_layout.scss]: Should these be here --`_layout.scss`? Border, background and color? My be move them into `_themes.scss`? :)

Comment (joneff) [reply packages/default/scss/chat/_layout.scss]: currently, per design, we want all buttons to be like that: to have no border, background and inherit color

File: packages/default/scss/chat/_layout.scss [hunk 2]
```diff
@@ -414,6 +414,61 @@ $message-group-spacing: calc( #{$chat-avatar-size} + #{$chat-avatar-spacing} );
...
+        padding: $chat-toolbar-padding-y $chat-toolbar-padding-x;
+        width: 100%;
+        box-sizing: border-box;
+        overflow: hidden;
+        position: relative;
+        flex: 0 0 auto;
+
+        .k-button-list {
+            // margin-bottom: -20px;
+            // padding-bottom: 20px;
```
Comment (gyoshev) [inline packages/default/scss/chat/_layout.scss]: Remove the commented out code, or add a comment why these lines are commented out.

Comment (joneff) [reply packages/default/scss/chat/_layout.scss]: will do

**Pull request merged**